src/main_control/src/main.py

```diff
@@ -9,34 +9,34 @@
 
 assert True # turn off this before race
 
-class AlphabetRecognize:
+# class AlphabetRecognize:
```

Comment:
Commenting these class declaration is not necessary. You should only comment line `139` if you want to disable the functionality of `alphabetNode`.